sympy/sets/__init__.py

```diff
@@ -3,3 +3,7 @@
 from .fancysets import ImageSet, Range, ComplexRegion
 from .contains import Contains
 from .conditionset import ConditionSet
+
+from ..core.singleton import S
```

Comment:
There's something specifically wrong with the way Sphinx is importing things. Or possibly we have some issue in our conf.py.